ui/src/app/shared/components/array-input/array-input-field.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
```

Comment:
I've sent PR which refactors array input: removes duplicated state and reduce duplication: https://github.com/alexec/argo-cd/pull/10 . Let me know what you think please.